src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -41,6 +43,27 @@
 -include_lib("couch_mrview/include/couch_mrview.hrl").
 
 
+get_local_purge_doc_id(Sig) ->
+    Version = "v" ++ config:get("purge", "version", "1") ++ "-",
+    ?l2b(?LOCAL_DOC_PREFIX ++ "purge-" ++ Version ++ "mrview-" ++ Sig).
+
+
+get_value_from_options(Key, Options) ->
+    case couch_util:get_value(Key, Options) of
+        undefined ->
+            Reason = binary_to_list(Key) ++ " must exist in Options.",
+            throw({bad_request, Reason});
```

Comment:
If you search code base we usually return `{bad_request, binary()}`. This can be done as `Reason = <<"'", Key/binary, "' must exists in options.">>`

Reply:
changed in 407b1dc